src/main/java/org/spongepowered/common/config/category/EntityCollisionCategory.java

```diff
@@ -39,14 +39,10 @@
                                                          + "performance when there are more than 8 entities on top of each other such as a 1x1 \n"
                                                          + "spawn pen. Set to 0 to disable.")
     private int maxEntitiesWithinAABB = 8;
-    @Setting(value = "defaults", comment = "Default maximum collisions used for all entities/blocks unless overridden.")
```

Comment:
this is being used https://github.com/SpongePowered/SpongeCommon/blob/73bd5a690228331902862e20115ce9c841e68b70/src/main/java/org/spongepowered/common/mixin/entitycollisions/MixinBlock_Collisions.java

Reply:
No, it's using https://github.com/SpongePowered/SpongeCommon/blob/bbb238a3292f5d988dbd0c4e9773ab78f00d778b/src/main/java/org/spongepowered/common/config/category/CollisionModCategory.java#L39

Reply:
my bad